msticpy/analysis/outliers.py

```diff
@@ -32,6 +42,16 @@
         extra="ml",
     ) from imp_err
 
+try:
+    import rrcf
+except ImportError as imp_err:
```

Comment:
You might consider adding rrcf to the setup extra for ml
https://github.com/microsoft/msticpy/blob/8bc4652f016186325ee7fcbbc381850d2fe01239/setup.py#L47
Especially since you are indicating that this would be installed using `pip install msticpy[ml]`